Update desc for --no-progress (#157)
infection/site: src/guide/command-line-options.md

### `--no-progress`

This option has two effects:

> - It disables intermediate buffering of mutations used to count them. This causes progress bars to not have a total number of mutations displayed, while also reducing memory usage requirements and speeding up the entire process. It is benificial during CI, and for larger projects.

Author: ```suggestion
- It disables intermediate buffering of mutations used to count them. This causes progress bars to not have a total number of mutations displayed, while also reducing memory usage and speeding up the entire process. It is beneficial during CI, and for larger projects.
```

> - It disabled dynamic progress bars output to reduce the amount of generated text output.

Author: ```suggestion
- It disables dynamic progress bars output to reduce the amount of generated text output.
```

Disabling progress bars removes the following lines from output on Continuous Integration servers:

```bash
Processing source code files: 0/5678
Processing source code files: 1/5678
Processing source code files: 2/5678

> Processing source code files: 3/5678

Author: `Creating mutated files and processes` is gone

...
5k lines of text
...
Processing source code files: 5678/5678

```

> Progress bad display will be automatically disabled with or without this option when either `CI` or `CONTINUOUS_INTEGRATION` environment variables are set to `"true"`.

Reviewer: `bad` -> `bar`

> Progress bad display will be automatically disabled with or without this option when either `CI` or `CONTINUOUS_INTEGRATION` environment variables are set to `"true"`.

Reviewer: ```suggestion
Progress bar display will be automatically disabled with or without this option when either `CI` or `CONTINUOUS_INTEGRATION` environment variables are set to `"true"`.
```

> Progress bad display will be automatically disabled with or without this option when either `CI` or `CONTINUOUS_INTEGRATION` environment variables are set to `"true"`.

Author: This is closer to the truth. I think it is better to not force this option on users, for time being.
